crates/gitbutler-core/src/remotes/controller.rs

```diff
@@ -14,22 +14,22 @@ impl Controller {
         Self { projects }
     }
 
-    pub async fn remotes(&self, project_id: &ProjectId) -> Result<Vec<String>, Error> {
-        let project = self.projects.get(project_id)?;
+    pub async fn remotes(&self, project_id: ProjectId) -> Result<Vec<String>, Error> {
+        let project = self.projects.get(&project_id)?;
```

Comment:
@Byron I changed the argument to be type `ProjectId`, but it seems like `&ProjectId` is fairly deeply rooted throughout the application. Do we need to change these to just `ProjectId` at some point?
<img width="393" alt="Screenshot 2024-05-29 at 09 55 01" src="https://github.com/gitbutlerapp/gitbutler/assets/13354155/1f65e44a-5274-4c0c-9d10-259e19143699">

Reply:
> My stance here is that ideally we don't proliferate certain issues

100% Agree with you there

I'm still picking up rust best practices so I very much appreciate your reviews 👍